docs/design/autoware-architecture/planning/index.md

```diff
@@ -1,5 +1,83 @@
+  - [_no_stopping_area_](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_planner/no-stopping-area-design/)
+  - [_virtual_traffic_light_](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_planner/virtual-traffic-light-design/)
+  - [_occlusion_spot_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/#occlusion-spot)
+  - [_run_out_](https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_planner/run-out-design/)
+- [_obstacle_avoidance_planner_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/obstacle_avoidance_planner-design.ja/): calculate path shape under obstacle and drivable area constraints
+- [_surround_obstacle_checker_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/surround_obstacle_checker-design/): keeps the vehicle being stopped when there are obstacles around the ego-vehicle. It works only when the vehicle is stopped.
+- [_obstacle_stop_planner_](https://autowarefoundation.github.io/autoware.universe/main/planning/obstacle_stop_planner/): When there are obstacles on or near the trajectory, it calculates the maximum velocity of the trajectory points depending on the situation: stopping, slowing down, or adaptive cruise (following the car).
+  - [_stop_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/#obstacle-stop-planner_1)
+  - [_slow_down_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/#slow-down-planner)
+  - [_adaptive_cruise_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/#adaptive-cruise-controller)
+- [_costmap_generator_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/parking/costmap_generator/): generates a costmap for path generation from dynamic objects and lane information.
+- [_freespace_planner_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/parking/freespace_planner/): calculates trajectory considering the feasibility (e.g. curvature) for the freespace scene.
+- [_scenario_selector_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/scenario_selector/) : chooses a trajectory according to the current scenario.
+- [_motion_velocity_smoother_](https://tier4.github.io/autoware.iv/tree/main/planning/scenario_planning/common/motion_velocity_smoother/motion_velocity_smoother-design/): calculates final velocity considering velocity, acceleration, and jerk constraints.
```

Comment:
Should these links be replaced by the autoware.universe links ?
E.g., https://autowarefoundation.github.io/autoware.universe/main/planning/motion_velocity_smoother/

Reply:
Thanks. I fixed it.